api/src/main/resources/workbench-api.yaml

```diff
@@ -4439,10 +4439,6 @@ definitions:
         type: boolean
         default: false
         description: Whether to enable Multiple Cohort Review feature in Cohort Review
-      enableDrugWildcardSearch:
-        type: boolean
-        default: false
-        description: Whether to enable wildcard search for Durgs in Cohort Builder
```

Comment:
Story only says to remove references to flag, but not the flag itself. Please revert.